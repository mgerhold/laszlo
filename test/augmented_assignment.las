let n = 10;
n += 5;
println(n);
n *= -2;
println(n);
n /= 2;
println(n);
n -= 2;
println(n);

n += 2 + 2;
println(n);

let numbers = [1, 2, 3];
numbers += [4, 5, 6];
println(numbers);

let text = "Hello";
text += ", world!";
println(text);
