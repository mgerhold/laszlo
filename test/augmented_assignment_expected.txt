15
-30
-15
-17
-13
[1, 2, 3, 4, 5, 6]
Hello, world!